<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="Bt_CamClose">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>80</width>
     <height>62</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Sans Serif&quot;;
background-color: rgb(255, 89, 60);</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_CamOpen">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>80</width>
     <height>62</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
font: 18pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_SaveImg">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>80</width>
     <height>62</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: oblique 18pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>62</y>
     <width>80</width>
     <height>8</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>202</y>
     <width>80</width>
     <height>8</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>10</width>
     <height>272</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_SWB">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>80</width>
     <height>62</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: oblique 18pt &quot;Sans Serif&quot;;
background-color: rgb(0, 170, 127);
</string>
   </property>
   <property name="text">
    <string>SWB</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>132</y>
     <width>80</width>
     <height>8</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_wbShow">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: oblique 12pt &quot;Sans Serif&quot;;
background-color: rgb(187, 255, 235);
color: rgb(29, 127, 255);</string>
   </property>
   <property name="text">
    <string>WB Mode Tip</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>28</y>
     <width>390</width>
     <height>4</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_remove">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
font: 12pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
